src/engine/shared.rs

```diff
@@ -198,8 +198,10 @@ pub fn validate_name(name: &str) -> StratisResult<()> {
             format!("Name contains control characters : {}", name),
         ));
     }
-    let name_udevregex = Regex::new(r"[\$!]").expect("regex is invalid");
-    if name_udevregex.is_match(name) {
+    //let name_udevregex = Regex::new(r"[\$!]").expect("regex is invalid");
+    let name_udevregex = Regex::new(r"^[0-9A-Za-z#+-.:=@_]*$")
```

Comment:
Please leave the '/' in here, for completeness/ sake. It's fine if its redundant w/ the path checks below.